### ISSUE TITLE
VST3 plugins don't detect channel counts over 64

### ISSUE THREAD

**Issue by IokaCode** [leomccormack/SPARTA#83]
In Reaper running on my Windows machine, the VST3 versions are not detecting the 128 channels Reaper is providing, but are still showing me the message that the channel count needs to be higher. 7. Order is working still fine and VST2 version also up to 10. Order.

**Comment by leomccormack** [comment]
Yep. Although, this is not a limitation on our end. This is purely dictated by the VST3 SDK.
